### DISCUSSION TITLE
[Esame 3 giugno 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#454] [Esami M-Z]
![3 giu es.1](https://github.com/user-attachments/assets/4162d4b3-028e-4f77-a1da-307c7b37c38f)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/2c849c53-6f6c-4af5-a29f-da666bd216c1)
